setup.py

```diff
@@ -67,6 +67,7 @@
         'pytz',
         'semver',
         'uwsgi',
+        'tzdata==2023.3',
```

Comment:
@minhajuddin2510 Any particular reason to pin tzdata?  Shouldn't we allow it to get the latest as tz changes are added periodically?

Reply:
No particular reason, i removed it